movie/src/components/Navbar/index.jsx

```diff
@@ -27,57 +40,47 @@ const Navbar = () => {
 
   return (
     <Router>
-      <div className="navbar">
-        <img className="logo" src={LOGO} alt="" />
-
-        <ul className="leftNav">
-          <div class="dropdown">
-            <button class="dropbtn">Movie</button>
-            <div class="dropdownContent">
-              <NavbarPopup navbarList={mobileList} />
-            </div>
-          </div>
-
-          <div class="dropdown">
-            <button class="dropbtn">TVShows</button>
-            <div class="dropdownContent">
-              <NavbarPopup navbarList={tvShow} />
-            </div>
-          </div>
-
-          <div class="dropdown">
-            <button class="dropbtn">People</button>
-            <div class="dropdownContent">
-              <li>PopularPeople</li>
-            </div>
-          </div>
-
-          <div class="dropdown">
-            <button class="dropbtn">More</button>
-            <div class="dropdownContent">
-              <NavbarPopup navbarList={more} />
-            </div>
-          </div>
-        </ul>
+      <div className="navHight">
+        <div className="navbar nav">
+          <img className="logo" src={LOGO} alt="NavbarLogo" />
 
-        <ul className="rightNav">
-          <li className="liNav">EN</li>
-          <li className="liNav">JoinTMDB</li>
-          <li className="liNav">
-            <SearchIcon />
-          </li>
-          <li
-            onClick={() => {
-              setIsMenuOpen(!isMenuOpen);
-            }}
-            className={isHumbergerIcon ? "showItems" : "hideItems"}
-          >
-            <DehazeIcon />
-          </li>
-          <li>
-            <HumbergerNavbar onMobile={isMenuOpen} />
-          </li>
-        </ul>
+          <ul className="leftSideMenu">
+            {navbarData.map((nav, i) => {
+              return (
+                <div key={i} class="dropdown">
+                  <button class="dropbtn">{nav.label}</button>
+                  <div class="dropdownContent">
+                    <NavbarPopup navbarList={nav.component} />
+                  </div>
+                </div>
+              );
+            })}
+          </ul>
+          <ul className="rightSideMenu">
+            {rightSideMenu.map((menu, i) => {
+              return (
+                <li key={i} className="liNav">
+                  {menu.label}
+                </li>
+              );
+            })}
+            <li
+              onClick={() => {
+                setIsMenuOpen(!isMenuOpen);
+              }}
+              className={isHumbergerIcon ? "showItems" : "hideItems"}
+            >
+              <DehazeIcon />
+            </li>
+            <li>
+              <HumbergerNavbar
+                rightSideMenu={rightSideMenu}
```

Comment:
say no to `rightSideMenu` 🚫

Reply:
it is on the right, bong.
I find another name, but not found yet (haha)

Reply:
Bong!
I decide to rename:
*** leftSideMenu to dropdownMenu because those menu are dropdown to select in order move to another page.
*** rightSideMenu to singleMenu because those menu just click on it and then go to another page.